### PR TITLE
[blog] Qdrant 1.12 Release

### PR DESCRIPTION
**Pull request by davidmyriel**
[PREVIEW](https://deploy-preview-1218--condescending-goldwasser-91acf0.netlify.app/blog/qdrant-1.12.x/)

### CODE REVIEW

File: qdrant-landing/content/blog/qdrant-1.12.x.md [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+        "scores": [
+            1.4063001, 1.2531, 1.1550001, 1.1359, 1.4063001, 
+            1.2218001, 1.2531, 1.2218001, 0.70239997, 0.6146, 0.6353, 
+            0.5093, 1.0990001, 1.0349001, 1.1359, 1.0553
+            ],
+        "ids": [1, 2, 3, 4, 5, 6, 7, 8]
+    }
+}
+```
+*To learn more about the distance matrix, read [**The Distance Matrix documentation**](https://qdrant.tech/).*
```
Comment (agourlay) [inline qdrant-landing/content/blog/qdrant-1.12.x.md:112]: in progress here https://github.com/qdrant/landing_page/pull/1221

File: qdrant-landing/content/blog/qdrant-1.12.x.md [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+            {"a": 8, "b": 2, "score": 1.1359},
+            {"a": 8, "b": 3, "score": 1.0553}
+        ]
+    }
+}
+```
+
+### Configuration - Offsets
+
+The `offsets` endpoint is another method of calculating the distance between points:
```
Comment (generall) [inline qdrant-landing/content/blog/qdrant-1.12.x.md:84]: Offsets and pairs are not difference methods of calculating the distance. It is simply two different formats of representing results. 

Pairs are more convenient for representing graphs, while offsets are native for defining CSR sparse matricies https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.csr_matrix.html#scipy.sparse.csr_matrix

File: qdrant-landing/static/blog/qdrant-1.12.x/distance-matrix.png
Comment (generall) [inline qdrant-landing/static/blog/qdrant-1.12.x/distance-matrix.png:1]: we might want to make different screenshot, so the tree won't be too entangled.

File: qdrant-landing/content/blog/qdrant-1.12.x.md [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+  - facet api
+---
+[**Qdrant 1.12.0 is out!**](https://github.com/qdrant/qdrant/releases/tag/v1.12.0) Let's look at major new features and a few minor additions:
+
+**Distance Matrix API:** Efficiently calculate pairwise distances between vectors.</br>
+**GUI Data Exploration** Visually navigate your dataset and analyze vector relationships.</br>
+**Faceting API:** Dynamically aggregate and count unique values in specific fields.</br>
+
+**Text Index on_disk:** Reduce memory usage by storing text indexing data on disk.</br>
+**Geo Index on_disk:** Offload indexed geographic data on disk for memory efficiency.
```
Comment (timvisee) [inline qdrant-landing/content/blog/qdrant-1.12.x.md:27]: While the parameter is named this way, I think it's a bit better readable without the underscore:

```suggestion
**Text Index on disk:** Reduce memory usage by storing text indexing data on disk.</br>
**Geo Index on disk:** Offload indexed geographic data on disk for memory efficiency.
```

File: qdrant-landing/content/blog/qdrant-1.12.x.md [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+{
+    "key": "size",
+    "exact": true
+}
+```
+This feature provides flexibility between performance and precision, depending on the needs of your application.
+
+*To learn more about faceting, read the [**Facet API documentation**](https://qdrant.tech/).*
+
+## Text Index on_disk Support
```
Comment (timvisee) [inline qdrant-landing/content/blog/qdrant-1.12.x.md:200]: Same here?

```suggestion
## Text Index on disk Support 
```

File: qdrant-landing/content/blog/qdrant-1.12.x.md [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+        "max_token_len": 20,
+        "lowercase": true,
+        "on_disk": true
+    }
+}
+```
+
+*To learn more about indexes, read the [**Indexing documentation**](/documentation/concepts/indexing/).*
+
+## Geo Index on_disk Support
```
Comment (timvisee) [inline qdrant-landing/content/blog/qdrant-1.12.x.md:227]: Same here?

```suggestion
## Geo Index on disk Support 
```

**Review by timvisee — COMMENTED** [review]
Just a nitpick, no other real suggestions.

File: qdrant-landing/content/blog/qdrant-1.12.x.md [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+
+**Use Case:** A retail company with 10,000 customers wants to segment them by purchasing behavior. Each customer is stored as a vector in Qdrant, but without a dedicated API, clustering would need 10,000 separate batch requests, making the process inefficient and costly.
+
+You can use this API to compute a **sparse matrix of distances** that is optimized for large datasets. Then, you can filter through the retrieved data to find the exact vector relationships that matter.
+
+In terms of endpoints, we offer two different formats to show results:
+- **Pairs** are simple, intutitive and ideal for graph representation.
+- **Offsets** are more complex, but also native when defining CSR sparse matrices.
+
+### Configuration - Pairs
```
Comment (generall) [inline qdrant-landing/content/blog/qdrant-1.12.x.md:46]:
```suggestion
### Output - Pairs
```

File: qdrant-landing/content/blog/qdrant-1.12.x.md [hunk 7]
```diff
@@ -0,0 +1,266 @@
...
+            {"a": 7, "b": 3, "score": 1.0990001},
+            {"a": 7, "b": 1, "score": 1.0349001},
+            {"a": 8, "b": 2, "score": 1.1359},
+            {"a": 8, "b": 3, "score": 1.0553}
+        ]
+    }
+}
+```
+
+### Configuration - Offsets
```
Comment (generall) [inline qdrant-landing/content/blog/qdrant-1.12.x.md:86]:
```suggestion
### Output - Offsets
```

**Review by generall — APPROVED** [review]
Merge after docs

**Review by timvisee — APPROVED** [review]

**Pull request merged**